pkg/engine/validation.go

```diff
@@ -800,3 +788,29 @@ func matchesException(policyContext *PolicyContext, rule *kyvernov1.Rule) (*kyve
 	}
 	return nil, nil
 }
+
+func shouldRespondException(ctx *PolicyContext, rule *kyvernov1.Rule, log logr.Logger) *response.RuleResponse {
+	// if matches, check if there is a corresponding policy exception
+	exception, err := matchesException(ctx, rule)
+	// if we found an exception
+	if err == nil && exception != nil {
+		key, err := cache.MetaNamespaceKeyFunc(exception)
+		// TODO: increase metrics
+		if err != nil {
+			log.Error(err, "failed to compute policy exception key", "namespace", exception.GetNamespace(), "name", exception.GetName())
+			return &response.RuleResponse{
+				Name:    rule.Name,
+				Message: "failed to find matched exception" + key,
```

Comment:
needs a space after "...exception" before we + key

Reply:
Thx for letting me know 👍